### ISSUE TITLE
EditorItem Editable="false" invalid

### ISSUE THREAD

**Issue by yongmingxia** [dotnetcore/BootstrapBlazor#117]
### Describe the bug

add Editable="false" on EditorItem   Can't hide

```
<ValidateForm Model="@Model" >
    <EditorForm TModel="PerformanceRequirementInfo" AutoGenerateAllItem="false">
        <FieldItems>
            <EditorItem @bind-Field="@context.ID" Editable="false" />
            <EditorItem @bind-Field="@context.IDX" Editable="false" />
            <EditorItem @bind-Field="@context.Type">
                <EditTemplate Context="value">
                    <Select SkipValidate="true" @bind-Value="@context.Type" Items="@Types" />
                </EditTemplate>
            </EditorItem>
            <EditorItem @bind-Field="@context.TestCaseID" />
            <EditorItem @bind-Field="@context.Maneuver">
                <EditTemplate Context="value">
                    <Textarea @bind-Value="@context.Maneuver" placeholder="Please input....."></Textarea>
                </EditTemplate>
            </EditorItem>
            <EditorItem @bind-Field="@context.CustomerExpection">
                <EditTemplate Context="value">
                    <Textarea @bind-Value="@context.CustomerExpection" placeholder="Please input....."></Textarea>
                </EditTemplate>
            </EditorItem>
            <EditorItem @bind-Field="@context.ProposerName" />
        </FieldItems>
        <Buttons>
            <Button Icon="fa fa-save" Text="Save" ButtonType="ButtonType.Submit" />
        </Buttons>
    </EditorForm>
</ValidateForm>

```

### Further technical details
- BootstrapBlazor version 5.9.3
- Include the output of `dotnet --info`
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version
- Net5

**Comment by ArgoZhang** [comment]
@yongmingxia thank you for contract us

first. please click the star on the top right page.
second. i will fix it ASAP

**Comment by ArgoZhang** [comment]
@yongmingxia 

hi, this issue is not repro. can you check it again? 
BTW: you can check the first sample in https://www.blazor.zone/editorforms

```
<EditorItem @bind-Field="@context.Education" Editable="false" />
```

it works well.

**Comment by yongmingxia** [comment]
> @yongmingxia
> 
> hi, this issue is not repro. can you check it again?
> BTW: you can check the first sample at https://www.blazor.zone/editorforms
> 
> ```
> <EditorItem @bind-Field="@context.Education" Editable="false" />
> ```
> 
> it works well.

Delete  AutoGenerateAllItem="false" is normal！！

AutoGenerateAllItem="false"  and Editable="false" 
Can not be used at the same time？

**Comment by ArgoZhang** [comment]
@yongmingxia thanks. I repro it. this is a bug. will be fixed it in the next release.

**Comment by ArgoZhang** [comment]
@yongmingxia released 5.10.1
